api/XMLDocument.inc

```diff
@@ -182,7 +190,7 @@ class XMLDocument {
    * @return string
    *   the XMLDocument as a string.
    */
-  public function saveXML() {
+  public function saveXml() {
```

Comment:
Should ensure that the things call this particular method are updated to have the right func name.